### PR TITLE
fix broken logger

### PR DESCRIPTION
**Pull request by ralphtheninja**
Fixes:

logger-is-not-a-function

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 const debug = require("debug")("apn");
 
 const credentials = require("./lib/credentials")({
-	debug
+	logger: debug
 });
 
 const config = require("./lib/config")({
-	debug,
+	logger: debug,
```
Comment (florianreinhart) [inline index.js]: In config.js the debug logger is accessed via `dependencies.debug`. Please also adjust config.js to use `logger` instead of `debug`.

Comment (ralphtheninja) [inline index.js]: Absolutely!

**Comment by ralphtheninja** [comment]
@florianreinhart Let me know if you want me to squash commits before merge.

**Comment by florianreinhart** [comment]
LGTM, merging

**Pull request merged**